src/datumaro/cli/commands/__init__.py

```diff
@@ -28,6 +28,8 @@
     "get_project_commands",
 ]
 
+deprecated = "[DEPRECATED, will be removed in 1.12]"
+
```

Comment:
We should also mark the project commands as deprecated.

Reply:
Fixed.